README.txt

```diff
@@ -1,13 +1,19 @@
-Installer: http://MrFeinberg.com/haikufinder-1.1.zip
+Files: http://MrFeinberg.com/haikufinder-1.1.zip or use github repo.
 
-* 
-*	REQUIRES NLTK, THE NATURAL LANGUAGE TOOLKIT
-*	http://www.nltk.org/download
-*
+Installation requires NLTK and its punkt package. First-time steps
+Unzip main file.
+pip install nltk
+python
+import nltk
+nltk.download('punkt')
+exit()
+python setup.py install
+
+A script to begin processing is located in \scripts
```

Comment:
A command line program, which scans standard input for haikus, can be found at scripts/findhaikus.